src/families/tron/bridge/js.js

```diff
@@ -363,29 +383,74 @@ const getTransactionStatus = async (a, t): Promise<TransactionStatus> => {
       ? BigNumber(0)
       : await getEstimatedFees(a, t);
 
-  const totalSpent = amount.plus(estimatedFees);
+  const totalSpent = amountSpent.plus(estimatedFees);
 
   if (estimatedFees.gt(0)) {
-    const formattedFees = formatCurrencyUnit(
-      getAccountUnit(account),
-      estimatedFees,
-      {
-        showCode: true,
-        disableRounding: true
-      }
-    );
+    const fees = formatCurrencyUnit(getAccountUnit(account), estimatedFees, {
+      showCode: true,
+      disableRounding: true
+    });
 
-    warnings.fee = new FeeRequired(`Estimated fees: ${formattedFees}`);
+    warnings.fee = new FeeRequired("Estimated fees", { fees });
```

Comment:
FeeRequired is not adapted error in here. fee requires wording is "Fees are required" ( https://github.com/LedgerHQ/ledger-live-desktop/blob/develop/static/i18n/en/app.json#L1423 ), here is seems you instead want a wording that express the fact some fees are going to be used which is usually not the case.

maybe you could introduce a new `UnexpectedFees` ?